Converter/EntityConverter.php

```diff
@@ -56,21 +57,31 @@ public function apply(Request $request, ParamConverter $configuration)
         $class   = $configuration->getClass();
         $options = $this->getOptions($configuration);
         $mode  = empty($options['mode']) ? CrudTransformer::MODE_RETRIEVE : $options['mode'];
+        $times = array();
 
         if (null === $request->attributes->get($name, false)) {
             $configuration->setIsOptional(true);
         }
 
+        $stopWatch = new Stopwatch();
+        $stopWatch->start('EntityConverter');
         $object = $mode == CrudTransformer::MODE_CREATE ? null : $this->findObject($class, $request, $options, $name);
+        $stopWatch->lap('EntityConverter');
+        $times['findObject'] = $stopWatch->getEvent('EntityConverter')->getDuration();
         if (empty($object) || $mode == CrudTransformer::MODE_UPDATE) {
             $data = $this->crudTransformer->getRequestData($request, $mode);
             $this->crudTransformer->convertArrayToEntityAndValidate($data, $class, $mode, $object);
             $this->crudTransformer->setIdsFromRequest($object, $request, $mode, !empty($options['generated_id']));
+            $stopWatch->lap('EntityConverter');
+            $times['findObject'] = $stopWatch->getEvent('crudTransformer')->getDuration();
```

Comment:
Why are you overwriting the findObject key each time? We should have three separate values to compare,